tox.ini

```diff
@@ -18,7 +18,7 @@ passenv = CI TRAVIS TRAVIS_*
 [testenv:pycodestyle]
 deps =
   flake8
-  pycodestyle
+  pycodestyle==2.3.1
```

Comment:
This needs to be fixed regardless of this PR. I'll take care of it. It will probably cause a conflict because our solution elsewhere has to just been take `pycodestyle` out altogether.